src/Network/Network.Test/ScenarioTests/PrivateLinkServiceTests.ps1

```diff
@@ -24,6 +24,15 @@ function Check-CmdletReturnType
     return $cmdletReturnTypes -contains $realReturnType;
 }
 
+<#
+.SYNOPSIS
+Gets the subscription ID from the recorded mock for a test
+#>
+function getSubscription
```

Comment:
This function is already defined in `Common.ps1`. No need to add it again. Sorry for confusion.